### PR TITLE
Improve documentation for GraphScene

### PR DESCRIPTION
**Pull request by aebkea**
## Motivation
The documentation for `coords_to_point()` and `point_to_coords()` under `manim/scene/graph_scene.py` describes each function as doing the opposite of what it actually does, causing confusion.

## Overview / Explanation for Changes
I've modified the documentation in each to ensure that the functions of them are accurately described. I also fixed the example for `coords_to_point()` so that it renders as intended.

## Oneline Summary of Changes

```
- Adjusted documentation for coords_to_point() and point_to_coords(). (:pr:`PR NUMBER HERE`)
```

## Acknowledgements
- [x] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)

## Reviewer Checklist
- [ ] Newly added functions/classes are either private or have a docstring
- [ ] Newly added functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] Newly added documentation builds, looks correctly formatted, and adds no additional build warnings
- [ ] The oneline summary has been included [in the wiki](https://github.com/ManimCommunity/manim/wiki/Changelog-for-next-release)

### CODE REVIEW

File: manim/scene/graph_scene.py
Comment (jsonvillanueva) [inline manim/scene/graph_scene.py:271]: I know this isn't something you have written, but while this PR touches this method, we may as well improve the rest of the class/method. Please read our [documentation guidelines](https://github.com/ManimCommunity/manim/wiki/Documentation-guidelines-(WIP)#types).
```suggestion
         x : :class:`float`
```

**Review by jsonvillanueva — CHANGES_REQUESTED** [review]
Thanks for the contribution @aebkea! This is a good start; however, we should improve this PR to fix other aspects of GraphScene's documentation. In particular, the parameters/return types. Please checkout our [wiki page](https://github.com/ManimCommunity/manim/wiki/Documentation-guidelines-(WIP)) (in particular the section on types).

File: manim/scene/graph_scene.py
Comment (aebkea) [reply manim/scene/graph_scene.py]: Something about my system configuration (working on Windows10) is preventing me from running make in manim/docs, so I've done the best I can without being able to see the rendered documentation.

Comment (naveen521kk) [reply manim/scene/graph_scene.py]: For the docs, you should have a look at https://github.com/ManimCommunity/manim/wiki/Documentation-guidelines-(WIP)#types, also you can get a preview using RTD. https://manimce--1076.org.readthedocs.build/en/1076/

File: manim/scene/graph_scene.py
Comment (behackl) [inline manim/scene/graph_scene.py:267]: In the way this is currently written, it will make "The graph is smaller than the scene." show up as the one-line documentation of this function. Suggestion for improvement:

```suggestion
        """Maps a pair of coordinates (with respect to the axes)
        to a point in the scene.

        The graph is smaller than the scene. Because of this, coordinates
        in the graph don't map to coordinates on the scene.

        This method returns a scaled coordinate for the scene,
        given coordinates that correspond to the graph.
```

File: manim/scene/graph_scene.py
Comment (behackl) [inline manim/scene/graph_scene.py:272]:
```suggestion
            The x coordinate with respect to the graph.
```

File: manim/scene/graph_scene.py
Comment (behackl) [inline manim/scene/graph_scene.py:275]:
```suggestion
            The y coordinate with respect to the graph.
```

File: manim/scene/graph_scene.py
Comment (behackl) [inline manim/scene/graph_scene.py:280]:
```suggestion
            Coordinates of a point in the scene.
```

File: manim/scene/graph_scene.py
Comment (behackl) [inline manim/scene/graph_scene.py:310]: Please use a similar one-line description as above.

File: manim/scene/graph_scene.py
Comment (behackl) [inline manim/scene/graph_scene.py:563]:
```suggestion
            Can be any of "left", "right" or "center". Refers to where
```

File: manim/scene/graph_scene.py
Comment (behackl) [inline manim/scene/graph_scene.py:650]: This has a very long one-line summary. Also, rendering the reference seems not to work here (but I'm not sure why, I was under the impression that a space after the backtick is not necessary).

```suggestion
        """Returns a list of Riemann Rectangles.
        
        This method returns a list of multiple :class:`~.VGroup`s of Riemann
        Rectangles. The initial rectangles are relatively inaccurate,
```

File: manim/scene/graph_scene.py
Comment (behackl) [inline manim/scene/graph_scene.py:729]:
```suggestion
        """Transform between groups of Riemann Rectangles.

        This method is used to transform between two VGroups of Riemann Rectangles,
```

**Review by behackl — CHANGES_REQUESTED** [review]
Thank you for taking your time and going through this file to improve the documentation! I've added some minor language improvements and other comments; please have a look at them, apply them if you are satisfied or add a comment if you are not.

Other than that, I am very happy with this PR and think that we can merge it soon.

**Review by jsonvillanueva — APPROVED** [review]

**Comment by jsonvillanueva** [comment]
I reran the workflow for the black test @aebkea. It seems there's still an issue with the file and black. Please go to the root of your manim repository and run `black .` to run it on the entire project.

**Comment by aebkea** [comment]
> 
> 
> I reran the workflow for the black test @aebkea. It seems there's still an issue with the file and black. Please go to the root of your manim repository and run `black .` to run it on the entire project.
> 
> Aside from that, take a look at the other suggestions and thanks again for the contribution!

I ran `black .` from the root of the repository, and while it changed a bunch of files, it didn't change `graph_scene.py`. I just ended up manually making the changes which `black-check` seemed to expect.

File: manim/scene/graph_scene.py
Comment (behackl) [inline manim/scene/graph_scene.py:653]:
```suggestion
        This method returns a list of multiple :class:`VGroups <.VGroup>` of Riemann
```

File: manim/scene/graph_scene.py
Comment (behackl) [inline manim/scene/graph_scene.py:1119]:
```suggestion
        added_anims : Optional[Sequence[:class:`~.Animation`]]
```

**Review by behackl — COMMENTED** [review]
Two more suggestions to fix syntax errors, I'll apply them myself though and approve and merge this PR if the build succeeds.

**Review by behackl — APPROVED** [review]
Seems good to me now.

Thanks again for your contribution, @aebkea!

**Pull request merged**
